src/app/tests/TestReportingEngine.cpp

```diff
@@ -48,8 +48,35 @@ static SecureSessionMgr gSessionManager;
 static Messaging::ExchangeManager gExchangeManager;
 static TransportMgr<Transport::UDP> gTransportManager;
 static const Transport::AdminId gAdminId = 0;
+constexpr ClusterId kTestClusterId       = 6;
+constexpr EndpointId kTestEndPointId     = 1;
 
 namespace app {
+CHIP_ERROR ReadSingleClusterData(NodeId aNodeId, ClusterId aClusterId, EndpointId aEndPointId, FieldId aFieldId,
+                                 TLV::TLVWriter & aWriter)
+{
+    CHIP_ERROR err           = CHIP_NO_ERROR;
+    uint8_t effectIdentifier = 1;
+    uint8_t effectVariant    = 1;
+
+    VerifyOrExit(aClusterId == kTestClusterId && aEndPointId == kTestEndPointId, err = CHIP_ERROR_INVALID_ARGUMENT);
+
+    if (aFieldId == 0 || aFieldId == 1)
```

Comment:
could we add cohnstants for 0, 1 and 2 here? also what are the effectIdentifier and effectVariant above? Can they be const?